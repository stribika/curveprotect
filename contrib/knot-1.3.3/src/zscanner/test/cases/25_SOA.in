$ORIGIN	.
$TTL	1

; OK
@	SOA	@	@	0	0 0 0 0		; The simplest variant
@	SOA	tld.	tld	0	0 0 0 0 	; Absolute and relative dnames.
@	SOA	@	@	4294967295 0 0 0 0	; Maximal serial
@	SOA	@	@	0 4294967295 4294967295 4294967295 4294967295	; Maximal times
@	SOA	@	@	0	1d 1h 1m 1s	; Time units
@	TYPE6	\# 22 00 00 0000000000000000000000000000000000000000	; TYPE + Hexadecimal rdata
@	TYPE6	@	@       0       0 0 0 0		; TYPE
@	SOA	ns.tld. first\.second.tld. (		; Multiline record
	2007120710
	1w2d3h4m5s
	2h
	3m
	4s
)
@	soa	@	@       0       0 0 0 0		; Type in lower-case

; KO
@	SOA
@	SOA						; Empty rdata
@	SOA	\# 0					; Hex empty rdata
@	SOA	\#					; Missing hex length
@	SOA	@	@	1h	0 0 0 0		; Bad number
@	SOA	@	@	4294967296 0 0 0 0	; Serial overflow
@	SOA	@	@	0 4294967296 0 0 0	; Refresh overflow
@	SOA	@	@	0 0 4294967296 0 0	; Retry overflow
@	SOA	@	@	0 0 0 4294967296 0	; Expire overflow
@	SOA	@	@	0 0 0 0 4294967296	; Minimum overflow
@	SOA	@	@	0 0 0 0 0 x		; Unexpected item
@	SOA	\# 23 0001 00 0000000000000000000000000000000000000000	; Too long rdata (mname)
@	SOA	\# 23 00 0001 0000000000000000000000000000000000000000	; Too long rdata (rname)
@	SOA	\# 23 00 00 000000000000000000000000000000000000000000	; Too long rdata
@	SOA	\# 23 00 00   0000000000000000000000000000000000000000	; Bad rdata length
