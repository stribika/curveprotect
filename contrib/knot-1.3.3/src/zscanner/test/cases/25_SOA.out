OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=00 00 0000000000000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=03746C6400 03746C6400 0000000000000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=00 00 FFFFFFFF00000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=00 00 00000000FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=00 00 000000000001518000000E100000003C00000001
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=00 00 0000000000000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=00 00 0000000000000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=026E7303746C6400 0C66697273742E7365636F6E6403746C6400 77A23B46000C08A500001C20000000B400000004
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0006
RDATA=00 00 0000000000000000000000000000000000000000
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_ENUMBER32_OVERFLOW
------
WARNG=ZSCANNER_ENUMBER32_OVERFLOW
------
WARNG=ZSCANNER_ENUMBER32_OVERFLOW
------
WARNG=ZSCANNER_ENUMBER32_OVERFLOW
------
WARNG=ZSCANNER_ENUMBER32_OVERFLOW
------
WARNG=ZSCANNER_EBAD_REST
------
WARNG=ZSCANNER_EBAD_HEX_RDATA
------
WARNG=ZSCANNER_EBAD_HEX_RDATA
------
WARNG=ZSCANNER_EBAD_HEX_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
